### PR TITLE
mpvScripts.acompressor: Init at 0.35.1

### PR DESCRIPTION
**Pull request by nicoonoclaste**
###### Description of changes

- Add nicoo to `lib.maintainers`
- `mpvScripts.acompressor`: Init at 0.35.1

`acompressor` is a script that is part of `mpv`, but providing a separate
derivation makes it easier for users to enable that script in their
configuration (systemwide, through home-manager, or however else)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by nbraud** [comment]
PS: Fixed another failure, the version extracted from my config repo stil referred to `pkgs.mpv-unwrapped`

**Comment by nbraud** [comment]
Trivial rebase to solve conflict in `all-packages.nix`

File: pkgs/applications/video/mpv/scripts/acompressor.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenvNoCC, mpv-unwrapped, lib }:
```
Comment (mkg20001) [inline pkgs/applications/video/mpv/scripts/acompressor.nix:1]:
```suggestion
{ stdenvNoCC
, mpv-unwrapped
, lib
}:
```

File: pkgs/applications/video/mpv/scripts/acompressor.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+stdenvNoCC.mkDerivation rec {
+  pname = "mpv-acompressor";
+  version = mpv-unwrapped.version;
+  src = "${mpv-unwrapped.src.outPath}/TOOLS/lua/acompressor.lua";
+  dontBuild = true;
+  dontUnpack = true;
+  installPhase = ''
+    install -Dm644 ${src} $out/share/mpv/scripts/acompressor.lua
+  '';
+  passthru.scriptName = "acompressor.lua";
```
Comment (mkg20001) [inline pkgs/applications/video/mpv/scripts/acompressor.nix:12]:
```suggestion
stdenvNoCC.mkDerivation rec {
  pname = "mpv-acompressor";
  version = mpv-unwrapped.version;

  src = "${mpv-unwrapped.src.outPath}/TOOLS/lua/acompressor.lua";

  dontBuild = true;
  dontUnpack = true;

  installPhase = ''
    install -Dm644 ${src} $out/share/mpv/scripts/acompressor.lua
  '';

  passthru.scriptName = "acompressor.lua";
```

new lines make it more readable and are meant to reduce conflicts (esp the function arguments/dependencies)

**Review by mkg20001 — CHANGES_REQUESTED** [review]

**Comment by mkg20001** [comment]
>  Is nixpkgs's style documented anywhere?

Not really. I usually stick to the new lines as suggested. If in doubt, run nixpkgs-fmt on it. That makes the formatting mostly acceptable.

**Comment by mkg20001** [comment]
Also maintainers commit "maintainers: add nicoo" and you can write init all-lowercase while you're at it

**Comment by nbraud** [comment]
> Also maintainers commit "maintainers: add nicoo" and you can write init all-lowercase while you're at it

Done!  Is there a particular reason for lowercasing commit messages?

**Comment by mkg20001** [comment]
> > Also maintainers commit "maintainers: add nicoo" and you can write init all-lowercase while you're at it
> 
> Done! Is there a particular reason for lowercasing commit messages?

It's from the manual https://nixos.org/manual/nixpkgs/stable/#submitting-changes-making-patches

**Comment by mkg20001** [comment]
Ah so they *did* make coding conventions, just while I wasn't looking https://nixos.org/manual/nixpkgs/stable/#sec-syntax

**Pull request merged**